.github/workflows/tests.yaml

```diff
@@ -234,6 +234,7 @@ jobs:
               timeout-minutes: 45
               if: github.event_name == 'pull_request'
               run: |
+                  scripts/run_in_build_env.sh './scripts/build_python.sh --install_wheel build-env --include-yamltests'
```

Comment:
The CI issue is because you need `--include_yamltests` and not `--include-yamltests`

Additionally I would think you only need to modify line 204 from
`scripts/run_in_build_env.sh './scripts/build_python.sh --install_wheel build-env'`

to
`scripts/run_in_build_env.sh './scripts/build_python.sh --install_wheel build-env --include_yamltests`

Reply:
Thanks @tehampson! I missed the step at liine 204.